### DISCUSSION TITLE
thank you!

### DISCUSSION THREAD

**Discussion by calan1990** [denbkh/stoneshard-save-editor#8] [General]
just came back made a account and wanted to thank the creator dude ur a hero...! ive been waiting for character creation for so long and this pretty much does it im so fricking grateful thank u mate!

**Comment by TheAriesGuy** [comment]
> just came back made a account and wanted to thank the creator dude ur a hero...! ive been waiting for character creation for so long and this pretty much does it im so fricking grateful thank u mate!

hi calan, can you guide on how to edit my save file
